### ISSUE TITLE
Current Development Cycle or Roadmap

### ISSUE THREAD

**Issue by Qkessler** [altsem/gitu#299]
Hi there!

Congrats for building a cool product, really happy with it so far. I have been a long-time Magit user and have been recently trying to move to a terminal-based workflow.

I'm interested in the long-term maintenance of the project. Are you 
currently working in any feature/issue? Is there a particular ROADMAP potential contributors can see and help with?

Thanks!

**Comment by altsem** [comment]
Hey! I have had the question asked before. Currently there's no roadmap. But theres a bunch of open issues that might help.

Something in the back of my head regarding the project is to experiment with:
- support for autocomplete
- consolidate the ui code into "Widgets"

I had some peeks at termwiz/wezterm, and they seem to be adding some nicer support for this.

I should create tickets for this. Currently I'm on vacation til February. But id be happy to answer simpler things, and perhaps put some more effort into writing better issues.
Then I think its just a matter of giving a hint of priority (roadmap/pinning issues?).

**Comment by Qkessler** [comment]
Thanks so much for the response! Makes sense, take the time off and enjoy!

I'm in the process of implementing the `merge_menu` alongside all the merge commands. In my mind, having a Roadmap (that being through prioritized issues) is helpful for contributors to help out! Reduces the bus hit factor.

Best,
Quique.

**Comment by Qkessler** [comment]
Unrelated but what's your preferred way of communication? Say, a contributor has questions about specific direction. Is it through creating an issue?

**Comment by altsem** [comment]
@Qkessler I agree something like a roadmap to help paint a picture of what is being worked on would be helpful. I think a pile of issues alone is too nitty-gritty to get a full picture. I'll think about it!

As for communication channel, I like the idea of an open forum. There's the "discussions" section on GitHub nowadays.
Might want somewhere to chat too?
